cfme/tests/infrastructure/test_publish_vm_to_template.py

```diff
@@ -11,7 +12,9 @@
 
 pytestmark = [
     pytest.mark.provider([InfraProvider],
-    required_fields=[['templates', 'small_template']])
+    required_fields=[['templates', 'small_template']]),
+    test_requirements.vmware,
+    test_requirements.rhev
```

Comment:
@lina-nikiforova these test cases does test against RHV  right? I am adding the requirement.